features/managing_order_message/register_order_note.feature

```diff
@@ -0,0 +1,28 @@
+@order_message
+Feature: Register order note
+  In order to register a note on on the order detail page
+  As an Administrator
+  I want to have an appropriate form on Order view in menu
```

Comment:
I want to have an appropriate form on the order detail page

Reply:
DONE

Reply:
it's not changed

Reply:
DONE